Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1935,6 +1935,112 @@ final class LayerTests: XCTestCase {
     XCTAssertEqual(transformerTensor.shape, transformerResult.shape)
   }
 
+  func testGroupNorm() {
+    // The expected values were computed using the following Python code:
+    // ```
```

Comment:
Nit: markdown has no value outside of documentation (`///`) comments.  I suggest removing the backticks and adding 2 spaces after `//` on each line containing Python code; that will make it easier to read.  Actually I might start with `#!python`, just to clue in the reader who tends to see the "shape" of the comment before reading details.

Reply:
This is the standard style all other tests using.
Please see #499.